templates/_schemas/details/default.twig

```diff
@@ -2,14 +2,28 @@
 
 
 
+{% set graph = create('Spatie\\SchemaOrg\\Graph') %}
+{% set seomateMetaData = craft.seomate.getMeta({
```

Comment:
Maybe we should rename this variable to `entrySeoMeta` or `entrySeomateMeta` or something, for consistency?